stock_dashboard_api/views/stock_view.py

```diff
@@ -17,13 +19,28 @@ class StockView(MethodView):
     """
 
     def get(self, pk: int) -> Response:  # pylint: disable=C0103, R0201
-        """A method that return Stock if provided pk is valid
+        """A method that return Stock if provided pk is valid, or if provided valid pk, 'from' and 'to' in query string,
+         this method return list of stock data for some period of time.
 
         :param pk: Stock primary key (id)
         :return: Response with one Stock
         """
         if isinstance(pk, int):
```

Comment:
If you specify parameter type in URL, you can omit this check

Reply:
And if you still want to do this check, then do:
`if not isinstance(...):` - so you can raise some error, or something like this, can you don't have to raise indentation level for whole function boby, which makes it unreadable